app/migrations/0032_auto_20200131_0606.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 2.2.9 on 2020-01-31 06:06
+
+from django.db import migrations, models
```

Comment:
I'm a bit confused on why this migration exists. This shouldn't be part of this PR.